### PR TITLE
Use Ubuntu Core 20

### PR DESCRIPTION
**Pull request by AlanGriffiths**

### CODE REVIEW

File: RUNNING_IN_A_VM.md [hunk 1]
```diff
@@ -8,24 +8,23 @@ We need to be able to run the VM and, as the default build of qemu on Ubuntu doe
...
-
-[https://cdimage.ubuntu.com/ubuntu-core/18/stable/current/](https://cdimage.ubuntu.com/ubuntu-core/18/stable/current/)
+We also download an Ubuntu Core image from [https://cdimage.ubuntu.com/ubuntu-core/20/stable/current/](https://cdimage.ubuntu.com/ubuntu-core/20/stable/current/)
 
 Now, uncompress the image and move it to a convenient location:
 
-    $ unxz ~/Downloads/ubuntu-core-18-amd64.img.xz
-    $ mv ~/Downloads/ubuntu-core-18-amd64.img ~/snap/qemu-virgil/common/
+    $ unxz ~/Downloads/ubuntu-core-20-amd64.img.xz
+    $ mv ~/Downloads/ubuntu-core-20-amd64.img ~/snap/qemu-virgil/common/
```
Comment (Saviq) [inline RUNNING_IN_A_VM.md:16]: This shouldn't be necessary, `qemu-virgil` has `home` connected?

**Review by Saviq — CHANGES_REQUESTED** [review]
Small tweak - move to `~snap` unnecessary.

File: RUNNING_IN_A_VM.md (hunk 1 above)
Comment (AlanGriffiths) [reply RUNNING_IN_A_VM.md:16]: It does, but `~/Downloads/` seems an unfortunate place to recommend keeping a VM image.

**Review by Saviq — APPROVED** [review]
A'ight.

**Pull request merged**